client/question/services/FeedbackGeneratorService.js

```diff
@@ -324,7 +436,7 @@ tie.factory('FeedbackGeneratorService', [
      * @private
      */
     var _getRuntimeErrorFeedback = function(
-        codeEvalResult, rawCodeLineIndexes) {
+      codeEvalResult, rawCodeLineIndexes) {
```

Comment:
It's OK to leave this indented by two additional spaces (see reasoning in an earlier comment).

Reply:
As mentioned, indentation were changed to address linter errors. I will try to revert to original spacing (perhaps after addressing all the other comments and before final approval/merge).